CHANGELOG.md

```diff
@@ -0,0 +1,364 @@
+
+## 0.20.2
+
+- Fix regression in MD037
+- Improve MD038
+
+## 0.20.1
+
+- Fix regression in MD037
+
+## 0.20.0
+
+- Add `markdownlint-configure-file` inline comment
+- Reimplement MD037
+- improve MD005/MD007/MD013/MD018/MD029/MD031/MD034/MD038/MD039
+- Improve HTML comment handling
+- Update dependencies
+
+## 0.19.0
+
+- Remove support for end-of-life Node version 8
+- Add fix information for MD005/list-indent
+- Improve MD007/MD013/MD014
+- Deprecate MD006/ul-start-left
+- Add rationale for every rule
+- Update test runner and code coverage
+- Add more JSDoc comments
+- Update dependencies
+
+## 0.18.0
+
+- Add MD048/code-fence-style
+- Add fix information for MD007/ul-indent
+- Add `markdownlint-disable-file`/`markdownlint-enable-file` inline comments
+- Add type declaration file (.d.ts) for TypeScript dependents
+- Update schema
+- Improve MD006/MD007/MD009/MD013/MD030
+- Update dependencies
+
+## 0.17.2
+
+- Improve MD020/MD033/MD044
+
+## 0.17.1
+
+- Fix handling of front matter by fix information
+
+## 0.17.0
+
+- Add `resultVersion` 3 to support fix information for default and custom rules
+- Add fix information for 24 rules
+- Update newline handling to match latest CommonMark specification
+- Improve MD014/MD037/MD039
+- Update dependencies
+
+## 0.16.0
+
+- Add custom rule sample for linting code
+- Improve MD026/MD031/MD033/MD038
+- Update dependencies
+
+## 0.15.0
+
+- Add `markdownlint-capture`/`markdownlint-restore` inline comments
+- Improve MD009/MD013/MD026/MD033/MD036
+- Update dependencies
+
+## 0.14.2
+
+- Improve MD047
+- Add `handleRuleFailures` option
+
+## 0.14.1
+
+- Improve MD033
+
+## 0.14.0
+
+- Remove support for end-of-life Node version 6
+- Introduce `markdownlint-rule-helpers`
+- Add MD046/MD047
+- Improve MD033/MD034/MD039
+- Improve custom rule validation and in-browser demo
+- Update dependencies
+
+## 0.13.0
+
+- Improve MD013/MD022/MD025/MD029/MD031/MD032/MD037/MD041
+- Deprecate MD002
+- Improve pandoc YAML support
+- Update dependencies
+
+## 0.12.0
+
+- Add `information` link for custom rules
+- Add `markdownItPlugins` for extensibility
+- Improve MD023/MD032/MD038
+- Update dependencies
+
+## 0.11.0
+
+- Improve MD005/MD024/MD029/MD038
+- Improve custom rule example
+- Add CONTRIBUTING.md
+- Update dependencies
+
+## 0.10.0
+
+- Add support for non-JSON configuration files
+- Pass file/string name to custom rules
+- Update dependencies
+
+## 0.9.0
+
+- Remove support for end-of-life Node versions 0.10/0.12/4
+- Change "header" to "heading" per spec (non-breaking)
+- Improve MD003/MD009/MD041
+- Handle uncommon line-break characters
+- Refactor for ES6
+- Update dependencies
+
+## 0.8.1
+
+- Update item loop to be iterative
+- Improve MD014
+- Update dependencies
+
+## 0.8.0
+
+- Add support for using and authoring custom rules
+- Improve MD004/MD007/MD013
+- Add `engines` to `package.json`
+- Refactor
+- Update dependencies
+
+## 0.7.0
+
+- `resultVersion` defaults to 2 (breaking change)
+- Ad MD045
```

Comment:
Add

Reply:
Sorry about all those typos. Was clearly in a hurry to do this. Fixed everything.